models/flavor.js

```diff
@@ -19,16 +19,21 @@ function saveFlavor(flavorData) {
 }
 
 
-function getFlavors(filter) {
+function getAllFlavors(filter) {
```

Comment:
I think if this is explicitly meant to grab all flavors then this shouldn't accept a filter but should pass an empty object to `.find({})`